how-tos/merge-and-staging-instance.md

```diff
@@ -0,0 +1,10 @@
+# Staging and merging instance
+
+There is an AWS instance to run staging and merging instances so that we are fewer hops away from the hg repos.  
+
+To access it, you can either 
+1. Create an new instance with the AMI named "Mozilla release staging/merge machine".  I used a M3.large instance with 32 gb space added.
+2. Use the existing instance named "Mozilla release staging/merge machine"
+You will need a pem file to access them, the key for it is the usual location
```

Comment:
Might be worth adding more people's keys to it. So far I only see you, Johan and me but we should have at least all the people on releaseduty hook for 57.